modules/s2i/bash/artifacts/usr/local/s2i/run

```diff
@@ -17,14 +17,19 @@ if test -r "${JBOSS_CONTAINER_MAVEN_DEFAULT_MODULE}/scl-enable-maven"; then
     source "${JBOSS_CONTAINER_MAVEN_DEFAULT_MODULE}/scl-enable-maven"
 fi
 
-JAVA_OPTS="${JAVA_OPTS:-${JAVA_OPTIONS}}"
+if [ -z "${JAVA_OPTS+isunset}" ] && [ -n "${JAVA_OPTIONS+isset}" ]; then
+    JAVA_OPTS="$JAVA_OPTIONS"
+fi
```

Comment:
Please add a test for `JAVA_OPTIONS=foo` and `JAVA_OPTS` being unset, expecting `foo` being passed to the java command line.

Reply:
I've added that test, and it fails, now to work out why.